bee-node/bee-node/src/cli.rs

```diff
@@ -9,6 +9,10 @@ use structopt::StructOpt;
 /// Defines the node CLI arguments.
 #[derive(Clone, Debug, StructOpt)]
 pub struct NodeCliArgs {
+    #[structopt(short, long, help = "The listening port")]
+    port: u16,
```

Comment:
Not precise enough, we'll end up with many listening ports. Maybe use `gossip_port` instead ?